spec/stop_visits.schema.json

```diff
@@ -0,0 +1,250 @@
+{
+  "primaryKey": [
+    "date",
+    "trip_id_performed",
+    "stop_sequence",
+    "vehicle_id"
+  ],
+  "missingValues": [
+    "NA",
+    "NaN",
+    ""
+  ],
+  "fields": [
+    {
+      "name": "date",
+      "type": "date",
+      "description": "Service date. References GTFS",
```

Comment:
How does this reference GTFS? Which fields? I'm not sure how it would?

Reply:
Only indirectly via calendars.txt + calendar_dates.txt. In other words, you have to be able to find the service_id in the GTFS calendar tables for a `date`.

Reply:
The report indicated GTFS as an external reference here. Do we want to mention that it's an indirect reference in the description field (per @botanize's comment), or drop the mention of the GTFS reference entirely?

Reply:
Easiest thing to do is stick with the report but say it's indirect (via the non-materialized service calendar implied by the combination of calendar.txt and calendar_dates.txt). The connection only matters because (nearly) all of the other IDs are GTFS IDs, so a date must be in a valid service period covered by a feed, but that bar is so low and obvious that it seems like you couldn't possibly (I'm sure I'm wrong) have a trip performed on a date where the trip was defined in a feed but the service date wasn't.